src/runtime/runtime-debug.cc

```diff
@@ -1151,6 +1152,10 @@ RUNTIME_FUNCTION(Runtime_RecordReplayAssertExecutionProgress) {
     }
   }
 
+  if (gRecordReplayReportPerformanceEvents) {
+    recordreplay::PerformanceEvent(/* ScriptAdvanceProgress */ 13, nullptr, 0);
```

Comment:
Can we have an enum or should we move that to the Q1 "code sharing refactoring" docket?

Reply:
The idea behind this API is to make it easy to add new performance events without having to have a bunch of boilerplate to call new recorder functions or to add stuff to shared headers, that slows down development for everyone with long recompilations.  So, I'd rather keep things flexible for now.

Reply:
Adding shared code is hard right now. If the enum was in a new header file instead of getting piled onto existing files, it would not cause "a bunch of boilerplate" nor "slow down development". Feels like code design and architecture decisions are going to be an important topic in the near future.